besu/src/test/java/org/hyperledger/besu/cli/BesuCommandTest.java

```diff
@@ -1329,6 +1330,16 @@ public void natMethodPropertyDefaultIsAuto() {
     assertThat(commandErrorOutput.toString()).isEmpty();
   }
 
+  @Test
```

Comment:
Can you add a test with a non default value and check the value is used ?

Reply:
Can you add a test that would trigger the exception raised by `validateNatParams` in `BesuCommand` ?

Reply:
Done